src/main/scala/ai/eto/rikai/sql/model/mlflow/MlflowCatalog.scala

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright 2021 Rikai authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ai.eto.rikai.sql.model.mlflow
+
+import ai.eto.rikai.sql.model.mlflow.MlflowCatalog.{
+  ModelFlavorKey,
+  TrackingUriKey
+}
+import ai.eto.rikai.sql.model.{Catalog, Model, SparkUDFModel}
+import org.apache.spark.SparkConf
+
+import scala.collection.JavaConverters._
+
+/** Use MLflow as a persisted backend for Model Catalog
+  */
+class MlflowCatalog(val conf: SparkConf) extends Catalog {
+
+  private val mlflowClient = new MlflowClientExt(conf.get(TrackingUriKey))
+
+  /** Create a ML Model that can be used in SQL ML in the current database.
+    */
+  override def createModel(model: Model): Model = {
+    val name = mlflowClient.createModel(
```

Comment:
How does it work with mlflow.

Does it mean that for each registered model, it will create a separate model in MLflow?

Reply:
Yes, I think so, each line in `show models` corresponds to the newest version of each model in Mlflow.